.pre-commit-config.yaml

```diff
@@ -86,6 +86,8 @@ repos:
             test_speechManager/speechManagerTestHarness.py
           )
         )
+    - id: check-illegal-windows-names  # Added hook
+    - id: requirements-txt-fixer  # Added hook
```

Comment:
can you make these comments more descriptive?
It looks like `requirements-txt-fixer` has messed up the file still, perhaps best to remove it?

Reply:
Can you help assist with modifications here?

Reply:
@dpy013 we can take on this work later